breadth-first-traversal/README.md

```diff
@@ -0,0 +1,11 @@
+## Breadth First Traversal
```

Comment:
No points lost for this since it was never a requirement, but it would be good to include instructions for a user to `npm i` any extra dependencies if your code depends on another npm module.